src/main/java/uk/gov/companieshouse/service/AppealService.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.companieshouse.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.exception.AppealNotFoundException;
+import uk.gov.companieshouse.model.Appeal;
+import uk.gov.companieshouse.model.CreatedBy;
+import uk.gov.companieshouse.repository.AppealRepository;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+
+@Slf4j
+@RequiredArgsConstructor
+@Service
+public class AppealService {
+
+    private final AppealRepository appealRepository;
+
+    public String saveAppeal(String companyId, Appeal appeal, String userId) throws Exception {
+
+        appeal.setCreatedBy(CreatedBy.builder().id(userId).build());
```

Comment:
Mutating method input arguments is usually not a great idea cause in Java arguments are passed as references. I'd rather see immutable objects in use and copies being made in mutation is necessary.

Could be tech debt due to removal in the future.